Tests/Common/DIGraphTests.swift

```diff
@@ -0,0 +1,11 @@
+@testable import Common
+import Foundation
+import SharedTests
+import XCTest
+
+class DIGraphTests: IntegrationTest {
+    func testDependencyGraphComplete() {
+        let graph = DIGraph.getInstance(siteId: testSiteId)
+        graph.testDependenciesAbleToResolve() // test will fail if an exception occurs while running this function
+    }
+}
```

Comment:
This code is copy/pasted into each of the modules of this project (except the `import` statements are unique). It's required that each module have their own copy of this test function instead of just running the test function once in one of the modules (such as `Common`). 

My only idea at this time to prevent this copy/paste is to autogenerate this test function so each time the dependency injection graph is generated, this test gets generated.